features/subtraction.feature

```diff
@@ -0,0 +1,20 @@
+Feature: Subtraction
+  As a mathematician
+  I want my calculator to subtract numbers
+  So I can know their result
+
+  Scenario: Subtracting numbers to get a result
+    When I subtract two numbers
+    Then I get the result
```

Comment:
You shouldn't be testing that your get a result. The acceptance criteria is that you get an integer in return. Looking at your steps, you've implemented the correct test, but you have not described it properly.

No points taken off, just FYI.